doc/source/wsrep-status-index.rst

```diff
@@ -179,6 +179,12 @@ The sequence number of the current transaction in IST.
 
 The sequence number of the first transaction in IST.
 
+.. variable:: wsrep_last_applied
+
+   :Version: Introduced in 5.7.20-29.24
```

Comment:
you should use :rn: for the version (:rn:`5.7.20-29.24`), this way it will be linked to the release notes.